test/core/parser/str02.v3

```diff
@@ -7,4 +7,7 @@ class A {
 	var j: string = "`~!@#$%^&'*()-_=+";
 	var k: string = "[{]}|;:,<.>/?";
 	var l: string = "\r\t\n\"\'\\\x4C";
+	var m: string = "--你好，世界！--";
```

Comment:
Can you split this test out to a separate file (so tests are additive)?

Reply:
Sure.